packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -638,9 +638,11 @@ class ListTile extends StatelessWidget {
     this.onTap,
     this.onLongPress,
     this.selected = false,
+    this.autofocus = false,
```

Comment:
How about also adding `focusColor`, `hoverColor`, and `focusNode` here, for parity with other widgets? You'd just have to pass them to the `InkWell`, so there's nothing to implement, really.

You're welcome to crib the tests for the colors from here: https://github.com/flutter/flutter/pull/53888/files#diff-557cb6cb0346a07994b565cb2a524544R1191

Reply:
Also, of course, if you don't have time for that, I'm happy to just add them in mine.

Reply:
Yes, could you please add them in yours?  It might take me a while to add those to all 4 classes.

Reply:
Sure, no problem.